server/constants/featureFlags.ts

```diff
@@ -18,14 +20,14 @@ export const Features = {
 export const featureFlags: FeatureFlags = {
   [Features.Adjudications]: {
     enabled: true,
-    allowedPrisons: ALLOW_ALL_PRISONS,
+    allowedPrisons: [prisonAgencyIds.Erlestoke],
   },
   [Features.Transactions]: {
     enabled: true,
-    allowedPrisons: ALLOW_ALL_PRISONS,
+    allowedPrisons: [prisonAgencyIds.Erlestoke],
   },
   [Features.Visits]: {
     enabled: true,
-    allowedPrisons: ALLOW_ALL_PRISONS,
+    allowedPrisons: [prisonAgencyIds.Erlestoke],
```

Comment:
I guess we need Approved Visitors in the profile before it can go live? 🤔 It's turned off at the moment but will want turning on.

Reply:
`Features.Visits` encompasses social/approved visitors so I think we're all good here 😄